source/Plugins/ExpressionParser/Swift/SwiftREPL.cpp

```diff
@@ -548,19 +548,22 @@ int SwiftREPL::CompleteCode(const std::string &current_code,
   if (swift_ast) {
     swift::ASTContext *ast = swift_ast->GetASTContext();
     swift::REPLCompletions completions;
-    SourceModule repl_module_info;
-    repl_module_info.path.push_back(ConstString("repl"));
-    swift::ModuleDecl *repl_module =
-        swift_ast->GetModule(repl_module_info, error);
-    if (!repl_module) {
-      repl_module = swift_ast->CreateModule(repl_module_info, error);
+    static ConstString g_repl_module_name("repl");
+
+    swift::ModuleDecl *repl_module = nullptr;
+    if (m_swift_repl_initialized)
+      repl_module =
+        swift_ast->GetModule(g_repl_module_name, error);
```

Comment:
Are LHS and RHS from two different branches?

Reply:
This should be rebased now.